tests/behat/features/component.feature

```diff
@@ -4,100 +4,16 @@ Feature: Automated Listing component in Landing Page
   Ensure that Landing Page content has the expected automated listing component.
 
   @api @nosuggest
-  Scenario: The content type has the expected component (and labels where we can use them).
```

Comment:
@asarfara cc @anthony-malkoun why this test is removed? Tide Automated Listing has no use without Tide API and this test ensures that the configuration from the automated listing is exposed correctly via JSON:API. It tests the features provided by the module hence it should stay in the module instead of tide landing page.